src/openrct2/ride/coaster/JuniorRollerCoaster.cpp

```diff
@@ -1622,9 +1622,10 @@ static constexpr const uint32_t junior_rc_track_pieces_diag_60_deg_down_to_25_de
 
 void junior_rc_paint_track_flat(
     paint_session* session, ride_id_t rideIndex, uint8_t trackSequence, uint8_t direction, uint16_t height,
-    const TileElement* tileElement, JUNIOR_RC_CHAINTYPE chainType)
+    const TileElement* tileElement, JuniorRcChainType chainType)
 {
-    uint32_t imageId = junior_rc_track_pieces_flat[chainType][direction] | session->TrackColours[SCHEME_TRACK];
+    uint32_t imageId = junior_rc_track_pieces_flat[static_cast<uint8_t>(chainType)][direction]
+        | session->TrackColours[SCHEME_TRACK];
```

Comment:
```suggestion
    uint32_t imageId = junior_rc_track_pieces_flat[EnumValue(chainType)][direction]
        | session->TrackColours[SCHEME_TRACK];
```

Reply:
Yeah I can totally do that - definitely agree that it's better. I had no idea that existed, but that's neat!